rollup.config.js

```diff
@@ -0,0 +1,55 @@
+import nodeResolve from 'rollup-plugin-node-resolve'
+import commonjs from 'rollup-plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import replace from 'rollup-plugin-replace'
+import uglify from 'rollup-plugin-uglify'
+import {sizeSnapshot} from 'rollup-plugin-size-snapshot'
+
+const input = './src/index.js'
+
+const name = 'jss'
+
+const getBabelOptions = () => ({
```

Comment:
why is it a function?

Reply:
can we use babelrc we already have?

Reply:
It's temporary I will tidy up babel configs in the next PR.